pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -4,10 +4,44 @@
 import pytest
 
 import pandas as pd
+from pandas.core import ops
 from pandas.core.sparse.api import SparseDtype
 import pandas.util.testing as tm
 
 
+@pytest.fixture(params=["integer", "block"])
+def kind(request):
+    return request.param
+
+
+@pytest.fixture(params=[True, False])
+def mix(request):
+    # whether to operate op(sparse, dense) instead of op(sparse, sparse)
+    return request.param
+
+
+@pytest.fixture(
+    params=[
+        operator.add,
+        ops.radd,
+        operator.sub,
+        ops.rsub,
+        operator.mul,
+        ops.rmul,
+        operator.truediv,
+        ops.rtruediv,
+        operator.floordiv,
+        ops.rfloordiv,
+        operator.mod,
+        ops.rmod,
+        operator.pow,
+        ops.rpow,
+    ]
+)
+def op(request):
+    return request.param
```

Comment:
does not all_arithmetic_operators do this?

Reply:
that passes the strings, which leads to different NotImplemented semantics

Reply:
then make a fixture which converts the existing strings to this if need be

this is duplicating things unnecessarily

Reply:
just moved the fixture to the top conftest, named `all_arithmetic_functions`.   we can't re-use the string one because that one doesnt include divmod and rdivmod.